### DISCUSSION TITLE
What is the best approach to choose "basis"

### DISCUSSION THREAD

**Discussion by gurasog** [Eelbrain/Eelbrain#67] [Q&A] [answered]
Hey!
Thank you very much for keeping this library updated and well documented!

I wanted to ask what should I rely on when I choose the value for "basis" parameter in "boosting" function. It seem it should be selected to somehow mimic the length of peaks in my original electrophysiological response, but I am not sure. 

Are there any theoretical implications or empirical rules how to do that?

Thanks a lot!

**Comment by christianbrodbeck** [comment, marked as answer]
Have you seen the section "Meta-parameter: Basis function" in https://doi.org/10.7554/eLife.85012?

For a specific problem you could use cross-validation to find the best parameter.
